packages/asc-ui/src/internals/Box/BoxWrapper.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react'
+import styled from '../../styled-components'
+
+export type Props = {}
+
+export const BoxWrapperStyle = styled.div`
+  position: relative;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  vertical-align: baseline;
+  font-family: 'AvenirNextLTW01-Regular', arial, sans-serif;
```

Comment:
Get font from theme?

Reply:
not needed, added extra info in the pull request.md